core/src/avm2/globals/flash/display/GraphicsPath.as

```diff
@@ -11,6 +11,7 @@ package flash.display {
             this.winding = winding;
         }
 
+        [API("674")]
```

Comment:
I think this one should be 672, for Flash Player 12.
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/display/GraphicsPath.html#cubicCurveTo()

Reply:
After triple-checking it, I just went ahead and fixed it. I hope you don't mind. If you think I'm wrong, feel free to open a follow-up PR, or yell at me on Discord, or similar.